t/05-scheduler-iso.t

```diff
@@ -64,4 +64,14 @@ is($schema->resultset("GruTasks")->search({taskname => 'download_asset'}), 0, 'g
 map { like($_, $expected, 'expected warning') } @warnings;
 is($schema->resultset("GruTasks")->search({taskname => 'download_asset'}), 1, 'gru task should be created');
 
+# Using non-asset _URL does not create gru job and schedule jobs
+output_like { $ids = OpenQA::Scheduler::Scheduler::job_schedule_iso(DISTRI => 'opensuse', VERSION => '13.1', FLAVOR => 'DVD', ARCH => 'i586', NO_ASSET_URL => 'nonexistent.iso') } qr/_URL downloading only allowed for asset types!/, $warning, 'expected debug info about non-asset type';
```

Comment:
Test::Warnings with the helper function `warnings` should mainly catch all warnings from perl not warnings we put in deliberately. I don't know if `warn` is better than `print STDERR`. As an alternative we should rather consider a proper logging framework.